src/logic/import/coco/COCOErrors.ts

```diff
@@ -0,0 +1,34 @@
+export class COCOAnnotationsLoadingError extends Error {
+    constructor(message) {
+        super(message);
+        this.name = "COCOAnnotationsLoadingError";
+    }
+}
+
+export class COCOFormatValidationError extends COCOAnnotationsLoadingError {
+    constructor(message) {
+        super(message);
+        this.name = "COCOFormatValidationError";
```

Comment:
ten `name` może możesz wyciągnąć tym: `this.constructor.name` - ale musisz sprawdzić czy pyknie jak zrobisz builda produkcyjnego - bo *uj wie czy to nie jest uglyfajowane